src/locales/en.json

```diff
@@ -0,0 +1,5 @@
+{
+  "harbours_list": {
```

Comment:
should we camelCase ?

Reply:
I don't know, any conventions we agreed on following?

Reply:
Not sure if we have convention on this, been camelCase in translation all my life :D Just asking